src/logic/safe/utils/buildSafeInformationUrl.ts

```diff
@@ -0,0 +1,7 @@
+import { getSafeServiceBaseUrl } from 'src/config'
```

Comment:
do you think we are going to use this function in another place besides `safeInformation.ts`?